gammapy/image/models/core.py

```diff
@@ -150,7 +150,7 @@ def __init__(self, lon_0, lat_0, sigma, frame="galactic"):
         )
         self.lat_0 = Parameter("lat_0", Latitude(lat_0), min=-90, max=90)
         self.sigma = Parameter("sigma", Angle(sigma), min=0)
-
+        self.tag = "SkyGaussian"
```

Comment:
I think we should define the tags as "class level" attributes, here is an example:
```python3
class A:
    a = 1  # class level attributes 
    def __init__(self, b):
        self.b = b  # instance level attribute 
```
The difference is, that class level attributes are shared across all instances of a model, while the instance attributes (such as parameter values...) can differ form instance to instance of the same model. The tag is certainly an attribute, that is shared by all instances. So one `SkyGaussian` does not have a different tag then another `SkyGaussian`. So just move the definition of the `tag` attribute up below the definition of the class name, just as in the example I've shown.